Show level input with range widget

hmpf opened the pull request:
I'm not happy with the styling. Dunno why it doesn't look as in the daisyUI docs.

hmpf committed:
Show level input with range widget

elfjes commented on src/argus_htmx/templates/htmx/incidents/_incident_filterbox.html:
```diff
@@ -16,6 +16,17 @@
                         <span class="label-text">{{ field.label }}</span>
                         {% if field|field_type == "booleanfield" %}
                             {{ field|add_class:"checkbox checkbox-accent border" }}
+                        {% elif field.name == "maxlevel" %}
```
not sure if this should be the criterium for showing a slider. The cleanest way would be to use a custom widget I think, Then it can be more easily reused for other filter parameters or forms.

but probably after the demo

hmpf replied:
I mentioned something similar in the newest commit: better to have widget templates (defined in the form widget in the form) than rely too much on widget-tweaks. After the demo :)

hmpf replied:
See #93 for an even better solution.

elfjes commented on src/argus_htmx/templates/htmx/incidents/_incident_filterbox.html:
```diff
@@ -16,6 +16,17 @@
                         {% if field|field_type == "booleanfield" %}
                             {{ field|add_class:"checkbox checkbox-accent border" }}
+                        {% elif field.name == "maxlevel" %}
+                            <div>
+                                {{ field|add_class:"range" }}
+                                <div class="flex w-full justify-between px-2 text-xs">
```
could benefit from some borders and probably  a `range-primary` class

elfjes reviewed (commented):
Looks nice, I like it :). A few comments below though

elfjes commented:
you say it doesn't look like the docs. This is how it looks for me. looks pretty similar I'd say?:

![image](https://github.com/user-attachments/assets/14040dc6-5a22-4fbe-87fb-025f1f355184)

podliashanyk reviewed (changes requested):
Tick marks should also be replaced with actual level numbers

hmpf committed:
Change tickmarks to level numbers

Note that the correct method is to have the range (1, 2, 3, 4, 5 in this
case) in the context. I suspect the best way of doing this is form-field
specific templates (set in the widget when defining the form class) and
moving away from widget-tweaks.

hmpf committed:
Uodate styles.css

hmpf commented:
Tick marks have been replaced with numbers in a hackish, ugly way and I don't think we should do it "properly" until after the demo. See #93.

podliashanyk reviewed (approved):
Works for me if we fix #93 after the demo. See a little suggestion below

hmpf committed:
Update src/argus_htmx/templates/htmx/incidents/_incident_filterbox.html

Co-authored-by: podliashanyk <60876078+podliashanyk@users.noreply.github.com>
